server/mentions_crawler_flask/blueprints/mention.py

```diff
@@ -11,6 +11,37 @@
 SORT_POPULAR_URL = "popular"
 SORT_RECENT_URL = "recent"
 
+# Get a set if mentions specified by a search string and page number
+@mention_bp.route("/search/<string:sort>/<int:page>", methods=["GET"])
+@authenticate()
+def search_mentions(user, sort, page):
+    search = request.args.get("search")
```

Comment:
would it be worth just having the search query be part of the route rather than using a parameter if it's only the one parameter, and since it's not optional anyways

So the route could be "/search/<string:query>/<string:sort>/<int:page>